datasets/paracrawl.yaml

```diff
@@ -0,0 +1,17 @@
+Name: Provision of Web-Scale Parallel Corpora for Official European Languages (ParaCrawl)
+Description: |
+ParaCrawl is a set of large parallel corpora to/from English for all official EU languages by a broad web crawling effort. State-of-the-art methods are applied for the entire processing chain from identifying web sites with translated text all the way to collecting, cleaning and delivering parallel corpora that are ready as training data for CEF.AT and translation memories for DG Translation.
```

Comment:
Please add two spaces before this line and I believe it'll be proper YAML syntax and the linting should pass. Thanks!